M2/InvertedIndexLoader.py

```diff
@@ -50,6 +50,6 @@ def getIndexDataAllTokens(tokens):
                 skippointer  = json.loads(l[-1]) # read skippointer first 
             from InvertedIndex import Postings
             data = [Postings.from_json(d) for d in json.loads(l[skippointer[t]])[t]]  # read only part of file where token is 
-            data_dict[t] = data 
-            prevc = currentc
+        data_dict[t] = data 
```

Comment:
Isn't this problematic?
Ex: for search "Design data", the code will store the same `data` variable for the second token as well.

Reply:
I unindented data part.

Reply:
Hmm, I think this should work now.

Reply:
Will merge